development/metamaskbot-build-announce.js

```diff
@@ -15,6 +15,21 @@ function capitalizeFirstLetter(string) {
   return string.charAt(0).toUpperCase() + string.slice(1);
 }
 
+function getHumanReadableSize(bytes) {
+  if (!bytes) {
+    return '0 Bytes';
+  }
+
+  const k = 1024;
+  const sizes = ['Bytes', 'KiB', 'MiB'];
+  const i = Math.floor(Math.log(Math.abs(bytes)) / Math.log(k));
```

Comment:
It looks like this doesn't handle large values well. This returns answers higher than `2` for large numbers, which would cause this to throw an error.

The single-character variable name isn't great either. Ideally we'd avoid those, using names that explain what the variable represents.

Maybe it would be clearer to do this in a more boring way, e.g.

```suggestion
  const kibibyteSize = 1024
  const magnitudes = ['Bytes', 'KiB', 'MiB'];
  let orderOfMagnitude = 0;
  if (bytes > kibibyteSize) {
    orderOfMagnitude = 1;
  } else if (bytes > 1000 *  kibibyteSize) {
    orderOfMagnitude = 2;
  }
```

Reply:
What do you mean by very large values? If we would add more magnitudes supported and then pass a number like `1_500_000_000_000` it would return `1.36 TiB`

Agreed on the variable names!

Reply:
The numbers work out fine for arbitrarily large values but we don't have unit descriptions past `MiB`, so it would say `undefined` as the unit. Better to have a large number of a defined unit.

I misspoke when I said it would "throw an error", it would not throw. Forgot for a moment that `undefined` coerces to a string just fine.

Reply:
Oh and that snippet is incorrect, my second condition should be `bytes > Math.pow(kibibyteSize, 2)`. I was mixing up the two formats.

Reply:
The `if` and `else if` should be swapped otherwise the magnitude will always be set to 1, also for MiB.
 I just committed the changes